src/proof_system/prover.rs

```diff
@@ -145,6 +147,30 @@ impl Prover {
         self.preprocessed_transcript.append_message(label, message);
     }
 
+    /// Adds the blinding scalars to a given vector. Always the same elements
+    /// of 'w_vec' are modified at the beginning of it, and appended at the end:
+    /// if hiding degree = 1: (b2*X(n+1) + b1*X^n - b2*X - b1) + w_vec
+    /// if hiding degree = 2: (b3*X^(n+2) + b2*X(n+1) + b1*X^n - b3*X^2 - b2*X
+    /// - b1) + w_vec
+    pub(crate) fn blind_poly<R: RngCore + CryptoRng>(
```

Comment:
What is the reason you add the `blind-poly` to the `Prover`? Doesn't it make more sense to have it added to the `fft::Polynomial` type?

Reply:
who blinds the polynomials is the prover, so I guess that the best option is to be a method of the prover

Reply:
Hmm, I'm not sure if I agree. I would say the method should be added to the type it operates on. The same way I also wouldn't add a `drive` method to the `Driver` but to the `Car` to give an example.